0xProject/0x-monorepo
# Use make-promises-safe as a preloader instead of manually importing

## Description



@fabioberger [pointed out](https://github.com/0xProject/0x-monorepo/pull/707#discussion_r196739496) that there's some inconsistency in whether and where we import the `make-promises-safe` package. What we really need to do is make sure that it is imported at least once (preferably exactly once, but there are no ill effects if it's imported multiple times) any time we run tests. That means we needed to add `import 'make-promises-safe';` to all of our test files.

Humans, myself included, are bad at remembering this sort of thing. By adding `make-promises-safe` as a pre-loader, [as described in the documentation](https://github.com/mcollina/make-promises-safe#as-a-preloader), we don't have to worry about remembering to add it to the top of every file. We still need to remember to add it to each new __package.json__.

## Testing instructions



Add an unhandled promise rejection (e.g. `Promise.reject(new Error('this should be caught'));`) to one of the test files and run it. You should observe that the process prints the error and then immediately exits (as opposed to the default unsafe behavior of just printing the error).

## Types of changes




## Checklist:
